Common++/header/Logger.h

```diff
@@ -153,7 +158,8 @@ namespace pcpp
 		 * @param[in] method The method in PcapPlusPlus code the log message is coming from
 		 * @param[in] line The line in PcapPlusPlus code the log message is coming from
 		 */
-		typedef void (*LogPrinter)(LogLevel logLevel, const std::string& logMessage, const std::string& file, const std::string& method, const int line);
+		typedef void (*LogPrinter)(LogLevel logLevel, const std::string &logMessage, const std::string &file,
+								   const std::string &method, const int line);
```

Comment:
In this case clang-format is using a combination of spaces and tabs, it's even more weird...